### DISCUSSION TITLE
Enforce entering the name of processor / working person?

### DISCUSSION THREAD

**Discussion by lsteinmann** [dainst/idai-field-client#182] [General]
This may be more of a human issue than anything else, but I have noticed a problem that sometimes annoys me more than it should: We have people working on the database that in some cases share a computer, and they (understandably) forget to change the name of the processor quite often. When at some point a resource-conflict arises, there is some confusion about who did what. Likewise, sometimes people (again, understandably) forget to enter their name at all when they install Field on a new machine and then suddenly, our most prolific team member is "anonymous". I tend to use this modified/created data from CouchDB to see who was working on what, so I actually see it quite often, which is not the case for most other people, I suppose. While I remind people to check this, it's not like I haven't forgotten it myself sometimes... it just happens, obviously.

In the end, not *that* many conflicts actually happen, so this may seem rather irrelevant at first. And maybe other projects are not in the same situation, where multiple people are working on the same resources. I would still argue that it is important. From a longterm-perspective, who and how many people worked on what when is rather standard and important metadata for really anything. The records show you exactly who worked in any way on a specific resource without people needing to remember that. You have a point called "Änderungshistorie Datensätze" in your Roadmap anyway (as well as user-management, which I think would be absolutely great...), but if this is ever to happen, we are currently actually producing a lot of very sad data that will be displayed then. 

Would it - as a quick and dirty fix - not be possible to somehow force people to enter their name on startup, or at least remind them? Is this a problem for anyone else at all? Are other people somehow less forgetful? Everything and more would be solved if there were a user management, where they would have to login with their personal accounts, obviously. Is that still actively on the Roadmap?

**Comment by friebsch** [comment]
Hi Lisa,

You're not the only one who is bothered by this. We have already opened an issue about this: 
https://github.com/dainst/idai-field/issues/151 

The name should be displayed prominently so that you can see who the current editor is and anonymous input should be prevented by a prompt

**Reply by lsteinmann** [reply]
Hey Fabian, 

maybe I should browse the issues a bit more diligently before I rant about something. ;) Sorry for the redundant post, then. That's great to hear, and it sounds like a good solution to just display the editor name somewhere so that people are able to notice!

**Comment by friebsch** [comment]
No problem. It's good to see that this is a concern that bothers other users

**Comment by dersmon** [comment]
The functionality is basically done and will be part of the next version. It will both bother users on startup if they still have "anonymous" as their name, and it will display the name beneath the project name in the top right corner.
